### DISCUSSION TITLE
Time remaining before clean filter

### DISCUSSION THREAD

**Discussion by antoweb** [ofalvai/home-assistant-candy#275] [Device support improvements]
Hello,
Someone can explain all this fileds what means?

{ "statusLavatrice":{ "WiFiStatus":"0", "Err":"0", "MachMd":"5", "Pr":"3", "PrPh":"0", "PrCode":"8", "SLevel":"0", "Temp":"40", "SpinSp":"14", "Opt1":"0", "Opt2":"0", "Opt3":"0", "Opt4":"0", "Opt5":"0", "Opt6":"0", "Opt7":"0", "Opt8":"0", "Opt9":"0", "Steam":"0", "DryT":"0", "DelVal":"435", "RemTime":"3540", "RecipeId":"0", "Lang":"0", "FillR":"0", "DisTestOn":"0", "DisTestRes":"0", "CheckUpState":"0", "T0W":"0", "TIW":"0", "T0R":"0", "numF":"0", "unbF":"0", "unbC":"0", "NtcW":"179", "NtcD":"9", "motS":"0", "APSoff":"0", "APSfreq":"62500", "chartL":"157242" } }

In particolar i need to know if i can extract the time remaining for clean filter like this image from the official app
![Screenshot_2023-06-06-22-56-29-751_it candy simplyfi-edit](https://github.com/ofalvai/home-assistant-candy/assets/32364871/d744d83c-eb6c-4529-9272-08ddc5cb4b44)

**Comment by ofalvai** [comment]
That's a good question, I never managed to figure out where this usage data comes from. If this is not coming from the device's response, then I guess the cloud service stores and calculates this data.
